duckduckgo/macos-browser
# Use latest privacy dashboard

Task/Issue URL:
Tech Design URL:
CC: @tomasstrba @mallexxx @samsymons 

**Description**:
Updates Privacy Dashboard with some small fixes
- Text can no longer be selected
- Parent entity searching matches for sites on subdomain

**Steps to test this PR**:
1. Go to `https://uk.yahoo.com`
1. Confirm that Privacy Dashboard detects this as owned by a tracker network, as it loads trackers from `yahoo.com` which is owned by Verizon Media

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
